src/Symfony/Component/Yaml/Tests/ParserTest.php

```diff
@@ -803,6 +803,84 @@ public function testMappingDuplicateKeyFlow()
         $this->assertSame($expected, Yaml::parse($input));
     }
 
+    /**
+     * @dataProvider getParseExceptionOnDuplicateData
```

Comment:
We don't *need* it, but it may help the one doing the cleanup when preparing 4.0...